woocommerce/woocommerce-ios
# Domain settings: minimal loading UI

Part of #8558 


## Description


In domain settings, we don't have a design for the loading state when fetching domains and site plan. As a minimal improvement, I added a redacted/shimmering state to the whole view. The main screen still looks empty because all the content require the fetched data, but at least the bottom CTA is showing a loading state.

## Testing instructions


Prerequisite: a WC store **with a WPCOM plan** is required

- Log in if needed, and continue with a WPCOM store
- Go to the Menu tab
- Tap on the settings CTA
- Tap the `Domain` row --> the bottom CTA should show a loading state 

## Screenshots


<img src="https://user-images.githubusercontent.com/1945542/216526788-29737cff-d277-4963-a90a-1c35dd32d3c4.png" width="300" />


---
